### ISSUE TITLE
Add button to copy evidence to clipboard

### ISSUE THREAD

**Issue by jrozner** [theparanoids/ashirt-server#428]
Not sure if this is something that will work cross os, cross browser, or make sense for all evidence types but it something that would be nice would be to have a button that copies at least images directly to the clipboard. When writing reports it's annoying to have to download each image and then drag it into program one at a time.

Not sure what makes the most sense in terms of UI. Maybe adding another option to the dropdown that already exists for options?

**Comment by JoelAtDeluxe** [comment]
To be clear, you want to but the binary data on the system clipboard to paste somewhere else?

**Comment by jrozner** [comment]
Need to dig into what the best option is and how it typically works for apps and different operating systems. Maybe it's a data uri or maybe it raw bytes or there's some other sort of metadata that goes along with it.

**Comment by JoelAtDeluxe** [comment]
There's some limitations here that might make this feature not as useful. Here's what I think the state of this feature is:

1. Most modern browsers support writing plain text to the clipboard. We already do this, and we can do this for text blocks pretty easily.
2. Chrome in particular supports this feature, but only for `image/png` mimetypes. That might be enough for this project, as most images are going to be screen captures, which are typically in png format. But, of course, any jpeg will fail.
3. Firefox seems to support this, but hides this ability behind a flag, and that flag defaults to off. Also, a casual test of this feature suggests that it too might be limited in what it can copy -- `image/jpeg` is not supported for example.
4. Needless to say, HAR format, even if copying were available, isn't likely to be paste-friendly. Likewise, the terminal recording format features a similar issue, and doesn't have a real mimetype

To recap:

| Browser | Can Copy Text | Can Copy PNG | Can copy anything else | 
| --- | --- | --- | --- |
| Chrome | Yes | Yes | No |
| Firefox | Yes | Yes¹ | No² |

¹ Support is locked behind a flag
² At least some formats aren't supported

Typescript also has some issues with this particular feature, though it's easy enough to turn off those flags.

**Comment by jrozner** [comment]
Gonna close this since the main use case, images, is already possible natively and it's not clear exactly how we would use this for other evidence types.
